core/runtime/bindings.rs

```diff
@@ -241,10 +256,13 @@ pub(crate) fn initialize_primordials_and_infra(
 }
 
 /// Set up JavaScript bindings for ops.
-pub(crate) fn initialize_deno_core_ops_bindings<'s>(
```

Comment:
This function needs to be split into two separate functions - one for "core" ops and one for "ext" ops.